sound/effect.h

```diff
@@ -231,9 +231,31 @@ class Effect {
     selected_++;
     if (selected_ == (int)files_found()) selected_ = 0;
   }
+#ifdef NO_REPEAT_RANDOM
+  int last_selected_;
```

Comment:
No, we are not going to add a new variable, we're going to fix how last_ works instead.

Reply:
no problem, will need last_ fixed as it rewrites between Select currently.

Reply:
so fix it?